glx/glxcmds.c

```diff
@@ -1672,16 +1671,12 @@ DoQueryContext(__GLXclientState * cl, GLXContextID gcId)
     sendBuf[9] = (int) (ctx->renderType);
 
     if (client->swapped) {
-        swaps(&reply.sequenceNumber);
-        swapl(&reply.length);
         swapl(&reply.n);
         SwapLongs(sendBuf, sizeof(sendBuf) / 4);
```

Comment:
this also needs to go away, because it's already swapped when written into the rpcbuf

Reply:
hmm, marking the line didn't work so well, so for clarification: 
talking about the SwapLongs() line.

Reply:
OTOH, I'm not 100% sure whether we can/should make the payload shorter. Right now, lots of the fields are just empty. Having the .n header field looks like the number of entries is variable.
We should check the spec about this.